test/pageobjects/page.js

```diff
@@ -9,12 +9,21 @@ module.exports = class Page {
    * Opens a sub page of the page
    * @param path path of the sub page (e.g. /path/to/page.html)
    */
-  open(path) {
-    return browser.url(`https://magento.softwaretestingboard.com/${path}`);
+  open() {
+    return browser.url('/');
```

Comment:
Do we need this step here?

Reply:
Since that method is removed to config file it is no longer needed.